spec/factories/applications.rb

```diff
@@ -60,13 +60,16 @@
       eligible_for_funding { true }
     end
 
+    trait :with_random_funding_cap_cohort do
+      cohort { association :cohort, :with_funding_cap, start_year: Faker::Number.between(from: 2024, to: 2029) }
```

Comment:
same here around adding set numbers, just setting the funding cap value to true should be enough to indicate that in a cohort